### ISSUE TITLE
Investigate background changes in Shoes 3

### ISSUE THREAD

**Issue by jasonrclark** [shoes/shoes4#1455]
[From the mailing list](http://lists.mvmanila.com/private.cgi/shoes-mvmanila.com/2017-July/000126.html)... see whether changes related to https://github.com/Shoes3/shoes3/issues/355 should cause updates in Shoes 4.

@ccoupe referenced seeing two bugs in trying to run this sample program on Shoes 4:

```
Shoes.app(title: " v1.0", width: 300, height: 500, resizable: true ) do
     background green
     @main = stack left: 0.05, top: 0.15, width: 0.9, height: 0.8,
scroll: true do
         background beige
         100.times do |i|
             para "#{i+1} times"
         end
     end
end
```

I'm betting one of them is [the fact it doesn't scroll since that's a long-standing thing we haven't gotten quite implemented yet](https://github.com/shoes/shoes4/issues/853). @ccoupe, other details you'd care to fill in about the other bad behavior you're seeing?

**Comment by ccoupe** [comment]
Correct. If you resize small a scroll bar does appear but it's not sized and the range is limited. 

FWIW, Shoes 3.3.3 will  recognize a `scroll: true` argument to `background` and moves the background with the contents scrolling. . Otherwise you get the traditional/old/correct-for-many behavior.
